testing.md

```diff
@@ -137,6 +137,9 @@ func TestTranslateTextWithConcreteClient(t *testing.T) {
 }
 ```
 
+To view a full working example see the
+[code here](https://github.com/googleapis/google-cloud-go/tree/master/internal/examples/fake).
```

Comment:
Consider moving this to the top. Also, we should make the link text more descriptive (mock vs fake etc.) for screen readers that skip from one link to the next.

Reply:
done

Reply:
Looks like the link text is still the same.

Reply:
Whoops, did move it to the top of the section but had forgotten to remove the old reference. Good catch!